### ISSUE TITLE
Install requirements via apt?

### ISSUE THREAD

**Issue by rovo89** [ClemensElflein/xcore.boot#2]
Wondering whether `apt install python3-tqdm` would be better than `pip install tqdm`?

**Comment by ClemensElflein** [comment]
I don't usually like installing using apt, because then the lib version is system dependent

It already has a requirements.txt, so it probably should be the usual mkvirtualenv + pip install -r requirements.txt

**Comment by rovo89** [comment]
Yep, that would be the best way. For system-wide installation, I understood that distribution packages should be preferred: https://veronneau.org/python-311-pip-and-breaking-system-packages.html

The README currently doesn't mention a version number, so the installation would be as random as using apt. With the requirements.txt, you get exactly what you've tested with.

To be clear, this has no priority, but if I didn't bring it up now, I'd forget about it.
